### PR TITLE
[fix] Use env var TESLA_VIN, ! VIN

### PR DESCRIPTION
**Pull request by baylanger**
Per the tesla-control help:

```
$ tesla-control --help 2>&1 |grep -B1 TESLA_VIN
  -vin string
    	Vehicle Identification Number. Defaults to $TESLA_VIN.
```

### CODE REVIEW

**Comment by baylanger** [comment]
Fixes following err:

```
Deploying public key to vehicle
Missing required flag: command requires a VIN
```

File: tesla_ble_mqtt/README.md [hunk 1]
```diff
@@ -45,4 +45,4 @@ Note that in both cases below, if you have already created a key pair that you w
 
 Full package originally built by https://github.com/iainbullock following exchanges on the Tesla Custom Integration Issues https://github.com/alandtse/tesla/issues/961#issuecomment-2150897886 
 
-The technical method was derived from Shankar Kumarasamy's blog https://shankarkumarasamy.blog/2024/01/28/tesla-developer-api-guide-ble-key-pair-auth-and-vehicle-commands-part-3
\ No newline at end of file
+The technical method was derived from Shankar Kumarasamy's blog https://shankarkumarasamy.blog/2024/01/28/tesla-developer-api-guide-ble-key-pair-auth-and-vehicle-commands-part-3
```
Comment (baylanger) [inline tesla_ble_mqtt/README.md:48]: This change is only an added new line at end-of-file (Unix vi editor).

File: tesla_ble_mqtt/rootfs/app/run.sh [hunk 2]
```diff
@@ -32,7 +32,7 @@ fi
 send_command() {
  for i in $(seq 5); do
   echo "Attempt $i/5"
-  tesla-control -ble -vin $VIN -key-name /share/tesla_ble_mqtt/private.pem -key-file /share/tesla_ble_mqtt/private.pem $1
+  tesla-control -ble -key-name /share/tesla_ble_mqtt/private.pem -key-file /share/tesla_ble_mqtt/private.pem $1
```
Comment (baylanger) [inline tesla_ble_mqtt/rootfs/app/run.sh:35]: With TESLA_VIN env var in place, the arg isn't needed anymore.

**Review by baylanger — COMMENTED** [review]
Unless you find something you want to change, looks like this PR is ready.

**Comment by raphmur** [comment]
Thanks for your contribution. Strange as the `-vin [whatever string]` should work.
It looks like the command changed is the one to send the key. I will change accordingly

**Pull request merged**